spec/index.html

```diff
@@ -939,6 +963,10 @@ <h3>RDF classes</h3>
           <td><a href="#ch_property">rdf:Property</a></td>
           <td>The class of RDF properties.</td>
         </tr>
+        <tr>
+          <td><a href="#ch_reificationproperty">rdf:ReificationProperty</a></td>
+          <td>The class of properties used for reification in RDF.</td>
+        </tr>
```

Comment:
This is no longer defined. Instead, rows for `rdf:reifies` and `rdf:Proposition` should be added.

Reply:
Yes, that's why I deleted this part.

Reply:
I still see `rdf:ReificationProperty` in two places of the document. Or am I missing something?